### DISCUSSION TITLE
Can we use tailwindcss along with Bootstarp?

### DISCUSSION THREAD

**Discussion by mohitpawar10** [tailwindlabs/tailwindcss#1683] [General]
Is it possible to use tailwind CSS along with Bootstrap? Some of the classes in the Bootstrap and Tailwind CSS are overlapped.

**Comment by adamwathan** [comment]
Yep, we created the `prefix` option to help avoid conflicts in these situations:

https://tailwindcss.com/docs/configuration/#prefix
